src/main/webapp/app/account/register/register.component.html

```diff
@@ -93,20 +93,21 @@ <h1>Registration</h1>
                 <div class="form-group">
                     <label class="form-control-label" for="confirmPassword">New password confirmation</label>
                     <input type="password" class="form-control" id="confirmPassword" name="confirmPassword" #confirmPasswordInput="ngModel" placeholder="Confirm the new password"
-                            [(ngModel)]="confirmPassword" minlength=4 maxlength=50 required>
+                            [(ngModel)]="confirmPassword" minlength=4 maxlength=50 required validateEqual="password">
                     <div *ngIf="confirmPasswordInput.dirty && confirmPasswordInput.invalid">
-                        <small class="form-text text-danger" *ngIf="confirmPasswordInput.errors.required"
-                              >
+                        <small class="form-text text-danger" *ngIf="confirmPasswordInput.errors.required">
```

Comment:
Hey,
Are changes like between line 98 just formatting changes?

Reply:
Yea some of them were just formatting changes to make the source code more consistent.